### ISSUE TITLE
Possible type mismatch on the ascending property

### ISSUE THREAD

**Issue by gwmccull** [Kingstinct/react-native-healthkit#2]
the types have `ascending` as a boolean but the Swift code has `ascending` as an `NSNumber`. When I try to pass `true`, it throws an error in some code that looks to be related to converting code in the RN bridge

sorry, I can't provide an exact error message. I was using the library for a hackathon project and the hackathon is now over and I've been moved to other work

my company may end up going forward with my hackathon project so it's possible I'll be back in that code at some point and I can help fix

**Comment by robertherber** [comment]
It would be great if you could show the snippet of code or at least which method this is related to. In most cases I think NSNumber should parse JS booleans nicely - so I don't know why it wouldn't in this case. Could you provide me with this example so I can have a look at it?

Hope your hackathon went well and this library helped a bit!

**Comment by gwmccull** [comment]
I'm pretty sure the code was something like this:

```
HealthKit.queryQuantitySamples(id, {
    from: cursor ? new Date(cursor) : subWeeks(new Date(), 1),
    to: now,
    ascending: true,
  })
```

I was having trouble with the app crashing and only figured it out when I randomly commented out the `ascending` property. In the final code, I changed it to `false` and it works, so it only seemed to be an issue with `true`. This was on MacOS Mojave with Xcode 11.3.1 running on the iPhone 11 Pro simulator with iOS 13

Other than that issue, the library was great and very helpful. I was excited to find one that was better maintained than the first couple that came up in searches

**Comment by robertherber** [comment]
Merged and released in 4.0.1
